habitat-baselines/habitat_baselines/config/experiments_hab3/pop_play_kinematic_oracle_humanoid_spot_fp.yaml

```diff
@@ -45,6 +45,9 @@ habitat:
       agent_1_oracle_nav_with_backing_up_action:
         motion_control: human_joints
         spawn_max_dist_to_obj: -1
+    measurements:
+      cooperate_subgoal_reward:
+        end_on_collide: False
```

Comment:
Just confirming, this is the correct value for general use of this config, and not specific to the HITL use case, correct?

Reply:
In this case it is specific to HITL. Because once the agents collide with each other the app crashes, because this is one of the episode over conditions and we cant execute actions if episode is over. 

Possible option here is to make this CLI configurable: add a flag that updates config and sets `end_on_collide: False` instead of updating the config.

Todo: decide how the Sandbox app should behave if agents collided and the episode is over.

Reply:
If it's specific to the sandbox app, we should probably do this as an override, right? Either via code or a readme instruction to add the override to the command line. I don't want to turn this config file into a sandbox-only config file that other folks are afraid to touch.